micrometer/src/test/java/org/jboss/eap/qe/micrometer/container/JaegerContainer.java

```diff
@@ -0,0 +1,112 @@
+package org.jboss.eap.qe.micrometer.container;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.client.WebTarget;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.jboss.eap.qe.micrometer.model.jaeger.JaegerResponse;
+import org.jboss.eap.qe.micrometer.model.jaeger.JaegerTrace;
+import org.jboss.eap.qe.ts.common.docker.Docker;
+
+public class JaegerContainer {
+    private static JaegerContainer INSTANCE = null;
+    public static final int DOCKER_CONTAINER_PORT_JAEGER_QUERY = 16686;
+    public static final int DOCKER_HOST_PORT_JAEGER_QUERY = DOCKER_CONTAINER_PORT_JAEGER_QUERY;
+    public static final int DOCKER_CONTAINER_PORT_JAEGER_OTLP = OpenTelemetryCollectorContainer.DEFAULT_OTLP_GRPC_PORT;
+    public static final int DOCKER_HOST_PORT_JAEGER_OTLP = DOCKER_CONTAINER_PORT_JAEGER_OTLP - 100;
+    private String jaegerEndpoint;
+
+    private final Docker jaeger;
+
+    private JaegerContainer() {
+        jaeger = new Docker.Builder("jaeger", "quay.io/jaegertracing/all-in-one:latest")
```

Comment:
Does it make sense to freeze the version? Maybe 1.58?

Reply:
Yeah, we're aiming at having latest for runtimes we test interoperability with.
But this is a tooling component so I do agree at freezing the version. Fixing, thanks.

Reply:
Thank you!